File: nicolaspuebla_proyecto_final/src/main/java/com/example/nicolaspuebla_proyecto_final/model/dto/LoginResponse.java
package com.example.nicolaspuebla_proyecto_final.model.dto;

import com.example.nicolaspuebla_proyecto_final.model.dataModels.MobileUser;

public class LoginResponse {
    private MobileUser user;
    private String token;

    public LoginResponse(){}

    public LoginResponse(MobileUser user, String token) {
        this.user = user;
        this.token = token;
    }

    public MobileUser getUser() {
        return user;
    }

    public void setUser(MobileUser user) {
        this.user = user;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }
}

File: nicolaspuebla_proyecto_final_android/app/src/main/java/com/example/nicolaspuebla_proyecto_final_android/data/model/auth/LoginResponse.kt
package com.example.nicolaspuebla_proyecto_final_android.data.model.auth

import com.example.nicolaspuebla_proyecto_final_android.data.model.dataClases.MobileUser

/**
 * Clase empleada para recibir la respuesta del microservicio tras un login.
 */
data class LoginResponse (
    val user: MobileUser,
    val token: String
)